src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLoggerScope.cs

```diff
@@ -78,6 +78,11 @@ public void EnrichAndCreateScopeItem(LogEvent logEvent, ILogEventPropertyFactory
                         destructureObject = true;
                     }
 
+                    if (key.StartsWith("$"))
+                    {
+                        key = key.Substring(1);
```

Comment:
The value would also have to be `ToString()`ed in order for this to take effect.

Reply:
sry, i was a little too fast. Fixed it again and added a unit test.